toptal/picasso
# fix(Helpbox): render Helpbox.Content as a div element

re #789

replacement for #816 after a discussion about the desired solution

### Description

To allow using block elements and Typography with `<p>` element inside `Helpbox.Content` render it as a `<div>`.

### How to test

- See it working on https://picasso.toptal.net/make-helpbox-content-a-div/?path=/story/components-folder--helpbox

### Review
